lerobot/configs/policy/act_real.yaml

```diff
@@ -0,0 +1,115 @@
+# @package _global_
```

Comment:
Let's just think about how many configs we want to maintain. We have configs that reproduce published SOTA metrics. This one does not do that. So we need to ask ourselves why we have it. I'm not fussed about what the outcome is. Just want to pose the question.